support-docs/README.md

```diff
@@ -244,6 +244,220 @@ The table below gives a description of the properties used to specify a switchin
 |package|Package definition of the switching regulator|Object| |
 |complianceList|List of compliances met by the switching regulator component|Array| |
 
+####  3.10.2	 Ldo

```

Comment:
The above section is already `3.10.2`, this should be `3.10.3`. The subsequent sections you added also need to be changed.

Reply:
Good call - fixed this.